Formula/ptpd.rb

```diff
@@ -0,0 +1,31 @@
+class Ptpd < Formula
+  desc "PTP daemon (PTPd) implementing the Precision Time Protocol (PTP) v2"
+  homepage "https://github.com/ptpd/ptpd"
+  url "https://github.com/ptpd/ptpd/archive/refs/tags/ptpd-2.3.1.tar.gz"
+  sha256 "267ad61d09d97069acec5d4878dceda20d0ddbebd27557d80230847848cee6c2"
+  license "BSD-2-Clause"
+  head "https://github.com/ptpd/ptpd.git", branch: "master"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  on_macos do
+    on_intel do
+      url "https://github.com/ptpd/ptpd.git", revision: "1ec9e650b03e6bd75dd3179fb5f09862ebdc54bf"
+    end
+    on_arm do
+      url "https://github.com/ptpd/ptpd.git", revision: "1ec9e650b03e6bd75dd3179fb5f09862ebdc54bf"
+    end
+  end
```

Comment:
this does look right, what is this for?

Reply:
These are workarounds to make the CI checks happy. This project is one that doesn't change very often and hasn't had a tagged release since 2015. There have been some commits to fix problems like the aforementioned U64 issue or newer architectures. So really I would like the homebrew version to use the latest commit on master for the release and tag it here as a 2.3.2 release candidate 0, but new formulas need be assigned a specific git tag. Since I don't own this project I cannot create a tag.

Reply:
If there is no stable tag Homebrew can not ship it.

Reply:
Ok I have contacted the maintainers to create a 2.3.2-rc0 tag, thanks